docs/index.html

```diff
@@ -209,6 +209,42 @@ <h1>Electronics and electrical design checklist</h1>
 				</li>
 			</ul>
 		</li>
+		<li>EMC
+			<ul>
+				<li> Requirements
+					<ul>
+						<li> What noise freqencies are emitted? (e.g. clock and SMPS frequencies) </li>
+						<li> What noise frequencies is the circuit sensitive to? (RF?) </li>
+						<li> What noise will be generated by connected devices? (e.g. mains adaptors) </li>
+						<li> What countries will the product be sold in? What standards cover these countries? </li>
+						<li> What is the end use for the product?
```

Comment:
```suggestion
						<li> Are there special <abbr title="Electromagnetic compatibility">EMC</abbr> requirements for the end-use?
```
I think this is a more clear prompt.

Reply:
Sort of agree, the special requirements come from the use case. Maybe both?
```suggestion
						<li> What is the end use for the product? Does this have any specific <abbr title="Electromagnetic compatibility">EMC</abbr> requirements ?
```

Reply:
Yeah, I think that will work too.